### DISCUSSION TITLE
Add 3D example and ResizableSplit

### DISCUSSION THREAD

**Discussion by CaramelHeaven** [ratatui/ratatui#1404] [Ideas]
1. Add example of making 3d, like in the picture below ?
2. Add ResizableSplit? 

<img width="658" alt="Screenshot 2024-10-06 at 02 17 59" src="https://github.com/user-attachments/assets/c8c1951e-32ee-4a53-b823-d2ed890d6c97">

Here is a repo https://github.com/ArthurSonzogni/FTXUI?tab=readme-ov-file

**Comment by joshka** [comment]
The first item (3D rendering) would be something I'd suggest putting in an external widget crate. It shouldn't really be something in the core library.

Can you expand on what you mean by the second? ResizableSplit
